docs/graphql/cloud/projects/create.rst

```diff
@@ -78,6 +78,8 @@ Hasura Cloud needs the following extra permissions:
 
 - (required) Read and write access to ``hdb_pro_catalog`` schema.
 
+Let's consider the following schema:
```

Comment:
This is not a schema description. We can remove this.

Reply:
okay